locations/jclouds/src/main/java/org/apache/brooklyn/policy/jclouds/os/CreateUserPolicy.java

```diff
@@ -110,6 +110,7 @@ public void run() {
     
     protected void addUser(Entity entity, SshMachineLocation machine) {
         boolean grantSudo = getRequiredConfig(GRANT_SUDO);
+        boolean resetPassword = machine.getConfig(SshMachineLocation.BYON_USER_RESET_PASSWORD);
```

Comment:
Is this addition still necessary given the similar change ("resetLoginPassword ") in 42be5897 by @bostko?